Tests/Simulation/IntegrationTestFixtureSim.cs

```diff
@@ -107,14 +120,15 @@ public override void SetupTestDevices()
                     DevAddr = DevAddr.Parse(deviceId.ToString("00000000", CultureInfo.InvariantCulture)),
                 };
 
-            TestDeviceInfo CreateOtaaDevice(int deviceId) =>
+            TestDeviceInfo CreateOtaaDevice(int deviceId, DeduplicationMode? deduplicationMode = null) =>
                 new TestDeviceInfo
                 {
                     DeviceID = deviceId.ToString("0000000000000000", CultureInfo.InvariantCulture),
                     AppEui = JoinEui.Parse(deviceId.ToString("0000000000000000", CultureInfo.InvariantCulture)),
                     AppKey = GetAppKey(deviceId),
                     IsIoTHubDevice = true,
                     SensorDecoder = "DecoderValueSensor",
+                    Deduplication = deduplicationMode is { } someDeduplicationMode ? someDeduplicationMode.ToString() : null
```

Comment:
nit: simplify with conditional access:

```suggestion
                    Deduplication = deduplicationMode?.ToString()
```